src/test/bundle_test.ts

```diff
@@ -291,12 +291,6 @@ suite('BuildBundler', () => {
         {
           root: path.resolve('test-fixtures/bundle-project'),
           entrypoint: 'index.html',
-          sources: [
```

Comment:
I just removed them because it turned out the sources bit of the config had no bearing on the test and it was misleading.  It was incidental and opportunistic, but not material.